src/helpers/helpers.canvas.js

```diff
@@ -153,7 +153,7 @@ export function drawPoint(ctx, style, radius, x, y, rotation) {
  * @private
  */
 export function _isPointInArea(point, area) {
-	var epsilon = 1e-6; // 1e-6 is margin in pixels for accumulated error.
+	const epsilon = 0.5;
```

Comment:
it'd be good to add a comment here that this is to match rounded decimals

Reply:
done